db/migrate/20250210155819_back_fill_decisions_approved_column.rb

```diff
@@ -0,0 +1,18 @@
+class BackFillDecisionsApprovedColumn < ActiveRecord::Migration[8.0]
+  def up
+    execute <<-SQL
+      UPDATE decisions
+      SET approved = CASE result
+        WHEN 0 THEN TRUE
+        WHEN 1 THEN FALSE
+      END;
```

Comment:
is result ever `nil`?

Reply:
result _shouldn't_ ever be `null` as there's a presence validation on result.

> just ignore those and leave as null?
the next migration makes the `approved` column non nullable so can't leave them as null! I'm hoping there's no decisions with a `null` result on production as it doesn't make sense to create a decision without a result!

Reply:
> I'm hoping there's no decisions with a `null` result on production
probs worth checking to prevent failed deploy